packages/nimble-components/docs/component-status.mdx

```diff
@@ -18,11 +18,11 @@ Here are components that are either implemented or on the roadmap for Nimble. So
 
 ### Legend
 
-| Icon | Description                                                                                                                                                              |
-| ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| ✅   | The component is ready for general use.                                                                                                                                  |
-| ⚠️   | The component is "[incubating](/docs/incubating-docs--docs)". These are not ready for general use and may make breaking changes without a "major" semantic version bump. |
-| ⭕   | The component doesn't exist yet. Please submit requests by commenting on the linked issue.                                                                               |
+| Icon                                                             | Description                                                                                                                                                              |
+| ---------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <nimble-icon-check severity="success"></nimble-icon-check>       | The component is ready for general use.                                                                                                                                  |
```

Comment:
Note: I tried to use the exported tag names here, but I couldn't get them to work correctly within an mdx file.

Reply:
You could import the files containing the elements to express the direct dependency and ensure the elements are available to be loaded. I guess it works without that because the story depends on those icon tags and the MDX depends on the story, but making the dependency explicit from here too seems slightly more correct to me.

`import '../src/icons/check';` etc